packages/sdk/odahuflow/sdk/models/feedback_model_feedback_request.py

```diff
@@ -10,21 +10,46 @@
 
 
 class FeedbackModelFeedbackRequest(Model):
-    """NOTE: This class is auto generated by the swagger code generator program.
+    """NOTE: This class is manually edited.
```

Comment:
@lev-titov I meant to write a model from scratch. Editing the generated model is not the best choice - it will be overridden by every single run of `make generate-python-client`. 
Moreover, in your current implementation, we obligate user to always have a feedback structure like `{"feedback": <actual_feedback_object>}`. While we want the feedback to be an arbitrary mapping, like a dict.

To write a model that actually is a custom mapping, I would use [Pydantic Custom Root Type feature](https://pydantic-docs.helpmanual.io/usage/models/#custom-root-types). 
```
>>> from typing import Any, Dict
>>> class MyModel(pydantic.BaseModel):    
...     __root__: Dict[str, Any]
...    
>>> MyModel.parse_obj({'test': {'some': 'feedback'}})
MyModel(__root__={'test': {'some': 'feedback'}})
>>> MyModel.parse_raw('{"test": {"some": "feedback"}}').json()
'{"test": {"some": "feedback"}}'
```

Just write your own models and use them in clients instead of generated ones.

**Alternatively**, we can simplify all this stuff and just operate with a `dict` as a feedback request payload instead of named data types. I don't mind at all. It is transparent and simple for users.